docs/howto/system_testing.md

```diff
@@ -302,12 +302,11 @@ There are some specific environment variables that should be leveraged to overco
 
 ### Kubernetes service deployer
 
-The Kubernetes service deployer requires the `_dev/deploy/k8s` directory to be present. It can include additional `*.yaml` files to deploy
-custom applications in the Kubernetes cluster (e.g. Nginx deployment). If no resource definitions (`*.yaml` files ) are needed,
-the `_dev/deploy/k8s` directory must contain an `.empty` file (to preserve the `k8s` directory under version control).
+The Kubernetes service deployer requires the `_dev/deploy/k8s` directory to be present. 
+It needs to include either a `kustomization.yaml` file or an `.empty` file. The first is needed to ensure the deployment of each resource.
```

Comment:
`It needs to include either a `kustomization.yaml` file or an `.empty` file. The first is needed to ensure the deployment of each resource.... **The second in case there are no resources to deploy.** `

I would maybe removed the following phrase since it is better expressed by the suggested statement above and it is redundant with what said before about the folder being needed anyway.